app/subscriber/src/features/home/home-filters/styled/HomeFilters.tsx

```diff
@@ -14,4 +15,17 @@ export const HomeFilters = styled(Row)`
     font-size: 0.5em !important;
   }
   font-family: Roboto', sans-serif;
+      button {
+      min-width: 5rem;
+      display: flex;
+      justify-content: center;
+      &.active {
+        background-color: ${(props) => props.theme.css.defaultRed} !important;
```

Comment:
`!important` means we're doing something wrong most of the time

Reply:
this is just moved code from another area, the next ticket on the list screen will remove this anyways (removed already on current branch)